resources/configs/common/patch_order

```diff
@@ -17,14 +17,12 @@ inox-patchset/0018-disable-first-run-behaviour.patch
 inox-patchset/0019-disable-battery-status-service.patch
 inox-patchset/0021-disable-rlz.patch
 inox-patchset/9000-disable-metrics.patch
-inox-patchset/9001-disable-profiler.patch
 
 debian/disable/promo.patch
 debian/disable/fuzzers.patch
 debian/disable/google-api-warning.patch
 debian/disable/device-notifications.patch
 debian/fixes/mojo.patch
-debian/fixes/crc32.patch
```

Comment:
I'm guessing these patches no longer applied cleanly (or at all)?

Reply:
That's correct
The file crc32c.h essentially upstreamed the changes from debian/fixes/crc32.patch in v63.
As for inox-patchset/9001-disable-profiler.patch, I removed that because it was removed in the v63 branch of the inox-patchset repo.
Most of the other inox-patchset changes in this PR come from my merging of the branch https://github.com/gcarq/inox-patchset/tree/feature/63 into the ungoogled chromium version as well

Reply:
Good to know. Thanks!